scene/3d/area_3d.cpp

```diff
@@ -605,6 +653,10 @@ void Area3D::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "gravity", PROPERTY_HINT_RANGE, "-32,32,0.001,or_lesser,or_greater"), "set_gravity", "get_gravity");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "linear_damp", PROPERTY_HINT_RANGE, "0,100,0.001,or_greater"), "set_linear_damp", "get_linear_damp");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "angular_damp", PROPERTY_HINT_RANGE, "0,100,0.001,or_greater"), "set_angular_damp", "get_angular_damp");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "wind_force_magnitude", PROPERTY_HINT_RANGE, "0,10,0.001,or_greater"), "set_wind_force_magnitude", "get_wind_force_magnitude");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "wind_attenuation_factor", PROPERTY_HINT_RANGE, "-3,0,0.001,or_lesser"), "set_wind_attenuation_factor", "get_wind_attenuation_factor");
+	ADD_PROPERTY(PropertyInfo(Variant::VECTOR3, "wind_source"), "set_wind_source", "get_wind_source");
+	ADD_PROPERTY(PropertyInfo(Variant::VECTOR3, "wind_direction"), "set_wind_direction", "get_wind_direction");
```

Comment:
For the area node, it would be more practical for the user to have just a `NodePath` property instead of entering the source and direction vectors by hand (the physics server parameters can stay the same though, since servers don't work with nodes).

You can set the property this way to restrict it to Node3D only:
`ADD_PROPERTY(PropertyInfo(Variant::NODE_PATH, "wind_source", PROPERTY_HINT_NODE_PATH_VALID_TYPES, "Node3D"), "set_wind_source", "get_wind_source");`
And then you can have just one member `wind_source_path` and set both source and direction from the `Node3D`.

What you can do then is to have a private method like `initialize_wind()` which sets the wind parameters in the physics server based on force, source and direction:
1. If force is < epsilon or source path is empty, just set wind force 0 and return
2. Get the `Node3D` node and set the source and direction based on its transform
```
Node3D *p_wind_source = Object::cast_to<Node3D>(get_node(p_wind_source_path));
ERR_FAIL_NULL(p_wind_source)
// Set force, source and direction in the physics server
```

The other thing to handle is that `get_node()` method can be called only when the node is already in the scene tree.

So in the wind source accessor, you can do this initialization only when the area node is in the scene (otherwise it would try to do it when the node is loaded but before it's added to the tree), checked this way:
`if (is_inside_tree()) { ... }`
That case in this accessor is for handling a change of wind source while the game is running.

And the second thing you would need is to do this initialization when entering the tree, in `Area3D::_notification` on `NOTIFICATION_ENTER_TREE`.

Reply:
per conversation, I'll address in a later PR